app/models/volume_mapping.rb

```diff
@@ -17,6 +17,22 @@ class VolumeMapping < ApplicationRecord
   supports_not :delete
   acts_as_miq_taggable
 
+  def mapped_to
+    host_initiator ? host_initiator : host_initiator_group
+  end
```

Comment:
This looks like something that should be handled by a subclass in your provider

Reply:
A VolumeMapping is meant to have either a host_initiator or a host_initiator_group. This holds for the base class. 
It's convenient to have a function that returns the item that it's mapped to.

What the reasoning in putting it in the subclass instead of the base class?